scripts/test_examples.js

```diff
@@ -8,28 +8,33 @@
 'use strict';
 
 const runCommand = require('./_runCommand');
-const getPackages = require('./_getPackages');
 
 const fs = require('graceful-fs');
 const path = require('path');
 const chalk = require('chalk');
 const mkdirp = require('mkdirp');
-const rimraf = require('rimraf');
 
-const EXAMPLES_DIR = path.resolve(__dirname, '../examples');
-const JEST_CLI_PATH = path.resolve(__dirname, '../packages/jest-cli');
+const ROOT = path.resolve(__dirname, '..');
+const BABEL_JEST_PATH = path.resolve(ROOT, 'packages/babel-jest');
+const EXAMPLES_DIR = path.resolve(ROOT, 'examples');
+const JEST_CLI_PATH = path.resolve(ROOT, 'packages/jest-cli');
 const JEST_BIN_PATH = path.resolve(JEST_CLI_PATH, 'bin/jest.js');
 const NODE_VERSION = Number(process.version.match(/^v(\d+\.\d+)/)[1]);
 const SKIP_ON_OLD_NODE = ['react-native'];
-const VERSION = require('../lerna').version;
+const INSTALL = ['react-native'];
 
-const packages = getPackages();
-
-const examples = fs.readdirSync(EXAMPLES_DIR)
+const examples = fs
+  .readdirSync(EXAMPLES_DIR)
   .map(file => path.resolve(EXAMPLES_DIR, file))
   .filter(f => fs.lstatSync(path.resolve(f)).isDirectory());
 
-function runExampleTests(exampleDirectory) {
+const link = (exampleDirectory, from) => {
+  const nodeModules = exampleDirectory + path.sep + 'node_modules' + path.sep;
+  mkdirp.sync(nodeModules);
+  runCommand('ln', '-fs ' + from + ' ' + nodeModules, exampleDirectory);
```

Comment:
That won't work if the user puts a space inside a folder name. An easy fix would be to change [this line](https://github.com/facebook/jest/blob/master/scripts/_runCommand.js#L18) to become:

```js
const callArgs = typeof args === 'string' ? args.split(' ') : args;
```

After that you would only have to pass `[ '-fs', from, nodeModules ]` instead of manually crafting the command line.